aiohttp/web.py

```diff
@@ -447,8 +447,9 @@ def run_app(app, *, host=None, port=None, path=None, sock=None,
                 pass
 
         try:
-            print("======== Running on {} ========\n"
-                  "(Press CTRL+C to quit)".format(', '.join(uris)))
+            if print:
```

Comment:
`if callable(print):` maybe? The discoverability of this argument is poor, intuition makes me want to use `print=True` which will fail miserably.

Reply:
better to "fail miserably" than fail silently. In neither case would `print=True` do what you expected.

Much better to use python typing hinting for public methods like this once 3.4 support is dropped.